packages/strapi-utils/lib/content-types.js

```diff
@@ -108,6 +112,58 @@ const isMediaAttribute = attr => {
   return (attr.collection || attr.model) === 'file' && attr.plugin === 'upload';
 };
 
+const getKind = obj => obj.kind || 'collectionType';
+
+const pickSchema = model => {
+  const schema = _.cloneDeep(
+    _.pick(model, ['connection', 'collectionName', 'info', 'options', 'attributes'])
+  );
+
+  schema.kind = getKind(model);
+  return schema;
+};
+
+const formatContentType = (
+  model,
+  { modelName, defaultConnection },
+  { apiName, pluginName } = {}
+) => {
+  if (apiName) {
+    Object.assign(model, {
+      uid: `application::${apiName}.${modelName}`,
+      apiName,
+      collectionName: model.collectionName || modelName.toLocaleLowerCase(),
+      globalId: getGlobalId(model, modelName, pluginName),
```

Comment:
not sure why you are passing pluginName here this would return undefined-something.

why not make the getGlobalId only a 2 param function & pass call it 
```js

getGlobalId(model, modelName) // for api 
getGlobalId(model, `${pluginNam}-${modelName}`) // for  plugin
getGlobalId(model, `admin-${modelName}`) // for admin